docs/user/versions.rst

```diff
@@ -148,7 +148,7 @@ a more stable version of the documentation available. Specifically:
 
 
 This feature is enabled by default on projects using the new beta addons.
-The beta addons can be enabled by using ``build.commands`` config key or via the new beta dashboard (https://beta.readthedocs.org) going to the admin section of your docs (:guilabel:`Admin` > :guilabel:`Settings`)
+The beta addons can be enabled by using ``build.commands`` config key or via the new beta dashboard (https://app.readthedocs.org) going to the admin section of your docs (:guilabel:`Settings` > :guilabel:`Addons (Beta)`)
```

Comment:
Are we still calling addons beta? Probably worth removing that as well.

Reply:
Good point. We are calling addons beta still, yes. I'm fine have a strong opinion about when remove the beta, but we should probably do it soon. After that, we should probably enable it by default for me projects. 

Then, continue trying to remove the old injection on older projects